service/interfaces.go

```diff
@@ -5,17 +5,19 @@ import (
 	"net/http"
 
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/dp-legacy-cache-api/api"
 	"github.com/ONSdigital/dp-legacy-cache-api/config"
 )
 
-//go:generate moq -out mock/initialiser.go -pkg mock . Initialiser
-//go:generate moq -out mock/server.go -pkg mock . HTTPServer
-//go:generate moq -out mock/healthCheck.go -pkg mock . HealthChecker
+// moq -out mock/initialiser.go -pkg mock . Initialiser
+// moq -out mock/server.go -pkg mock . HTTPServer
+// moq -out mock/healthCheck.go -pkg mock . HealthChecker
```

Comment:
why this change?

Reply:
go:generate  or go generate was not working for me. Maybe a misleading command? I am reading this line as call or execute this line of code in the command line. But I can change it back. The command to run in the shell is actually : moq -out mock/initialiser.go -pkg mock . Initialiser

Reply:
I thought this would introduce a linting error around comments, but it did not. Amended.

Reply:
I can't see that it's actually been amended?

Reply:
Sorry after reading i understand the command now. Go will actually try to run those tasks itself when you the go yacc tool installed - https://go.dev/blog/generate